src/parser/bison_parser.y

```diff
@@ -595,13 +640,34 @@ drop_statement:
 			$$->ifExists = false;
 			$$->name = $3;
 		}
+	|	DROP INDEX opt_exists index_name ON table_name {
+    			$$ = new DropStatement(kDropIndex);
+    			$$->name = $6.name;
+    			$$->ifExists = $3;
+    			$$->index_name = $4;
+    		}
 	;
 
 opt_exists:
 		IF EXISTS   { $$ = true; }
 	|	/* empty */ { $$ = false; }
 	;
 
+/******************************
+ * ALTER Statement
+ * ALTER TABLE students DROP COLUMN name;
+ ******************************/
+
+alter_statement:
+		ALTER TABLE table_name DROP COLUMN opt_exists column_name {
+			$$ = new AlterStatement(kAlterDropColumn);
+			$$->if_exists = $6;
```

Comment:
Multiple `IF EXISTS` can be present in one `ALTER TABLE` statement: one referencing the table and - in the case of `DROP COLUMN` - the other referencing the column. With the above rule, the latter is stored in the `it_exists` variable of the `AlterStatement`. However, this is not clear and the identifier of the variable should be more descriptive.

Reply:
Resolved in https://github.com/hyrise/sql-parser/pull/177/commits/3fe37ec388f707db6e35df0195f2f7643dd281f0